File: 2 - Condições/condicoes13.java
//Escreva um programa que leia 4 números e mostre o maior.

import java.util.Scanner;

public class condicoes13 {

  public static void main(String[] args) {
    System.out.println("Digite quatro números: ");
    Scanner scan = new Scanner(System.in);
    int num1 = scan.nextInt();
    int num2 = scan.nextInt();
    int num3 = scan.nextInt();
    int num4 = scan.nextInt();
    if (num1 > num2 && num1 > num3 && num1 > num4) {
      System.out.println("O maior dos números é: " + num1);
    } else if (num2 > num1 && num2 > num3 && num2 > num4) {
      System.out.println("O maior dos números é: " + num2);
    } else if (num3 > num1 && num3 > num2 && num3 > num4) {
      System.out.println("O maior dos números é: " + num3);
    } else {
      System.out.println("O maior dos números é: " + num4);
    }
  }
}

File: 2 - Condições/PHP/condicoes13.php
<?php

#Escreva um programa que leia 4 números e mostre o maior.

?>

<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8">
    <title>Condicionais 13 - PHP</title>
  </head>
  <body>
    <?php if (!empty($_GET)): ?>
      <?php if (($_GET['num1'] >= $_GET['num2']) && ($_GET['num1'] >= $_GET['num3']) && ($_GET['num1'] >= $_GET['num4'])): ?>
        <?= "O maior número é: " . $_GET['num1']; ?>
      <?php elseif (($_GET['num2'] >= $_GET['num1']) && ($_GET['num2'] >= $_GET['num3']) && ($_GET['num2'] >= $_GET['num4'])): ?>
        <?= "O maior número é: " . $_GET['num2']; ?>
      <?php elseif (($_GET['num3'] >= $_GET['num1']) && ($_GET['num3'] >= $_GET['num2']) && ($_GET['num3'] >= $_GET['num4'])): ?>
        <?= "O maior número é: " . $_GET['num3']; ?>
      <?php elseif (($_GET['num4'] >= $_GET['num1']) && ($_GET['num4'] >= $_GET['num2']) && ($_GET['num4'] >= $_GET['num3'])): ?>
        <?= "O maior número é: " . $_GET['num4']; ?>
      <?php endif; ?>
    <?php else: ?>
      <form action="condicoes13.php" method="get">
        Digite um número: <input type="text" name="num1"/> <br>
        Digite um número: <input type="text" name="num2"/> <br>
        Digite um número: <input type="text" name="num3"/> <br>
        Digite um número: <input type="text" name="num4"/> <br>
        <input type="submit" value="Enter">
      </form>
    <?php endif; ?>
  </body>
</html>
